### ISSUE TITLE
improvement : Highlight.php ?

### ISSUE THREAD

**Issue by ncou** [CouscousPHP/Couscous#262]
Hi,

Nice piece of code. I have just tested this tool but it seems to do the work. Simple and powerfull.

Just my 2 cents opinion, I think using the highlighter "https://github.com/scrivo/highlight.php" could be a fine plugin to Couscous, the idea is that most of the site using coucous should also use an highlighter (mostly in javascript to apply the highlight on the fly) so a plugin to enable the highlight during the conversion of markdow to html file could be a plus. And the user could simply add the highlight.js CSS file.

What do you think ?

**Comment by mnapoli** [comment]
Why not, but I'm afraid this would add some complexity and some more dependencies for something that can be solved in JS.

**Comment by ncou** [comment]
Yes it could add some complexity, and the JS way work fine. I just find the result a bit slow while doing the highlighting on-the-fly.
